compiler/core/snacc-validators.c

```diff
@@ -469,12 +469,14 @@ bool ValidateErrorsAreOfSameType(ModuleList* allMods)
 					if (GetROSEDetails(currMod, vd, NULL, NULL, &pszError, NULL, NULL, &errorType, false))
 					{
 						struct BasicType* errorBasicType = NULL;
-						bool bErrorIssue = false;
```

Comment:
The idea of the validation is that you can configure from external what you want to have checked.
I won´t allow to disable the checks in code because thats not the way they are indented to be used.